src/bridge/IrcHandler.ts

```diff
@@ -906,14 +906,14 @@ export class IrcHandler {
      * @return {Promise} which is resolved/rejected when the request finishes.
      */
     public async onMode(req: BridgeRequest, server: IrcServer, channel: string, by: string,
-                        mode: string, enabled: boolean, arg: string|null) {
+                        mode: string, enabled: boolean, arg: string|null): Promise<BridgeRequestErr|void> {
```

Comment:
Undefined makes more sense, yeah. At the risk of potentially ignoring a problem, I'm going to avoid diving into IrcEventBroker right now. It's fragile but works.